lib/services/sf.js

```diff
@@ -567,6 +567,13 @@ function StateAbstract(options)
         json: requestOptions.json,
         callback: async function (err, response, body)
         {
+          if (err && err.response) { // axios returns error for not 2xx responses - let's unwrap it
+            response = err.response;
+            err = null;
+            response.statusCode = response.status;
+            body = response.data;
+            response.body = response.data;
+          }
```

Comment:
What do you think about extract this unwraping to some commons?

Reply:
it's changing incoming parameters so it's not so easy - but eventually yes, we should have common method to normalize errors

Reply:
extracted to common place